src/app/code/community/Shopgate/Cloudapi/Model/Api2/Observers/WishlistsItemsRetrieve.php

```diff
@@ -49,4 +50,21 @@ public function execute(Varien_Event_Observer $observer)
         $output = $observer->getData('output');
         $output->setData('items', $items);
     }
+
+    /**
+     * @param Mage_Wishlist_Model_Item $item
+     */
+    private function appendChildId($item)
+    {
+        $buyRequestItem = new Varien_Object($item->getBuyRequest()->getData());
+        $superAttribute = $buyRequestItem->getSuperAttribute();
+        if (is_array($superAttribute) && count($superAttribute)) {
+            $parentProduct = Mage::getModel('catalog/product')->load($item->getProductId());
+            $childProduct  = Mage::getModel('catalog/product_type_configurable')->getProductByAttributes(
```

Comment:
You may need to do it for other products as well, our endpoints handle all products.

Reply:
do you mean for example a group product?

there a re the child information in the buyRequest like:

"super_group": {
                "548": "8",
                "551": "3"
            },

I can map this ids to the child_id, but in this case i sould rename this key to child_ids and return a array of items.

Reply:
Not a bad idea. Can you check the bundled as well? Please note that your tests are failing as you need to update them too, presumably.